spring-boot/admin/src/main/resources/application.yml

```diff
@@ -75,6 +75,31 @@ spring:
     rest:
       base-path: /api
 
+
+  security:
+    oauth2:
+      client:
+        registration:
+         # resource:
+          #  userInfoUri: https://api.github.com/user
+          github:
+            client-id: e188a243cc57ef90f2b5
+            client-secret: a5630e2f873890bdc8366aac0155e8113ece0997
+            scope[0]: user:email
+            scope[1]: read:user
+        #    accessTokenUri: https://github.com/login/oauth/access_token
+        #    userAuthorizationUri: https://github.com/login/oauth/authorize
+        #    tokenName: oauth_token
+            clientAuthenticationScheme: header
+            authenticationScheme: header
+            redirect-uri: http://localhost:8080/login/oauth2/code/github
+        # tell github that has a authentication name, the login attribute should be retrieved
```

Comment:
This comment is not really clear to me, what is the exact message? In any case, I assume it is related to line 99, so it should be moved there.

Reply:
Ok, so I moved the comment to the line 99.
Regarding your question, this comment meant that Spring should use the login attribute retrieved from the GitHub Oauth2 as the authentication name of the Security context (in this case, the login attribute is the Github username), I did this because by default, Spring Boot uses the Oauth token id as the authentication name.

Reply:
This was already solved on our CoatRack Internal Weekly TelCo and I applied your suggestion (check thread below)